docs/releases/2.0.txt

```diff
@@ -611,6 +611,13 @@ Miscellaneous
 
 * Support for GEOS 3.3.x is dropped.
 
+* The way data is selected for ``GeometryField`` is changed to improve
+  performance. If you don't already wrap geometry fields  in raw SQL queries
+  with ``SDO_UTIL.TO_WKBGEOMETRY()`` on Oracle or use ``::bytea`` on PostGIS,
```

Comment:
@timgraham, it was changed on MySQL and SpatiaLite too, but again, I don't think we need this "if" since I'd like to encourage people to use ``connection.ops.select`` so things will be more robust to changes in Django.

Reply:
Okay, I've amended it again.

Reply:
Thanks! Looks good now.